pgmpy/base/simple_causal_model.py

```diff
@@ -0,0 +1,163 @@
+from typing import Iterable, Optional, Union
+
+from pgmpy.base import DAG
+
+
+class SimpleCausalModel(DAG):
+    """
+    A specialized DAG class for simple causal models.
+
+    This class simplifies the creation of causal graphs commonly used in causal inference,
+    where the structure consists of exposures, outcomes, covariates (confounders), mediators, and instruments.
+    It automatically adds the standard edges:
+        - Exposures -> Outcomes (only if there are no mediators)
+        - Covariates -> Exposures
+        - Covariates -> Outcomes
+        - Instruments -> Exposures
+        - Exposures -> Mediators
+        - Mediators -> Outcomes
+
+    Diagram
+    -------
+    If mediators are present:
+        Covariates   Instruments
+           |             |
+           v             v
+        Exposures ---> Mediators
+           |              |
+           v              v
+        Outcomes <--------
```

Comment:
This doesn't look correct: Exposures to outcomes edges should not be present. A more standard way to draw these models is to place the instrument, exposure, mediator, and outcome in a single line.